src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.cs

```diff
@@ -32,42 +28,45 @@ internal partial class SourceNamespaceSymbol : NamespaceSymbol
         private int _flags;
 
         private LexicalSortKey _lazyLexicalSortKey = LexicalSortKey.NotInitialized;
+        private readonly string _injectedName;
```

Comment:
>private readonly string _injectedName; [](start = 8, length = 38)

Would it make sense to split this class into one that has only _mergedDeclaration ond one that has only _injectedName storage?